.github/workflows/npm-publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 15
```

Comment:
I'm not sure why such an old version was used in the publish script. Either it hasn't been updated, or publish with an older version could be preferable for some reason.